### PR TITLE
Bug 1223273 - Revise l10n string of fxa offline error message. r=yifan

### PR DESCRIPTION
**Pull request by danhuang1202**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=08c6f53e45cd873279e263844b86e0508692e899

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3c2bc3ee0b02efe5f198b6ba2bcb8f0823dc9d7b

File: tv_apps/browser/locales/browser.en-US.properties [hunk 1]
```diff
@@ -109,6 +109,8 @@ fxsync-sign-in-to-sync-description-plus=You can create a Firefox Account on your
 fxsync-desktop-bookmarks=Desktop bookmarks
 fxsync-mobile-bookmarks=Mobile bookmarks
 fxsync-enabled=Success! Your Firefox Account is ready to go! In the future, you can access and manage your account from Settings.
+fxsync-error-offline = Unable to Connect
+fxsync-error-offline-explanation = Please connect to a network from Settings.
```
Comment (begeeben) [inline tv_apps/browser/locales/browser.en-US.properties]: Please eliminate the spaces around the ```=``` sign.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8bc52a7c3fafbbb7604cac79693db70311679586

**Pull request merged**